backend/src/v5/routes/routesManager.js

```diff
@@ -40,7 +40,12 @@ const { modelTypes } = require('../models/modelSettings.constants');
 
 RoutesManager.init = (app) => {
 	const internal = app.get(BYPASS_AUTH);
+
 	app.use('/v5/teamspaces/:teamspace/projects', ProjectRoutes(internal));
+	app.use('/v5/teamspaces/:teamspace/projects/:project/containers', CreateModelGeneralRoutes(modelTypes.CONTAINER, internal));
+	app.use('/v5/teamspaces/:teamspace/projects/:project/federations', CreateModelGeneralRoutes(modelTypes.FEDERATION, internal));
```

Comment:
Why did you have to move federation/drawing?

Reply:
I did this because I wanted to achieve 100% coverge in tests because of this if statement

<img width="743" height="202" alt="image" src="https://github.com/user-attachments/assets/84574db8-51a0-44a3-b6c6-a505a76bf9d9" />

I could alternatively istanbul ignore this line

Reply:
@ChristosTsiotsias  yea let's istanbul ignore this 

In theory we don't need this if statement since it'll never be false the way we use it. but it's good to be there for clarity.